web/source/osk/inlinedOskView.ts

```diff
@@ -0,0 +1,182 @@
+// Includes KMW-added property declaration extensions for HTML elements.
+/// <reference path="../kmwexthtml.ts" />
+// Includes the touch-mode language picker UI.
+/// <reference path="languageMenu.ts" />
+/// <reference path="lengthStyle.ts" />
+// Defines desktop-centric OSK positioning + sizing behavior
+/// <reference path="layouts/targetedFloatLayout.ts" />
+/// <reference path="oskView.ts" />
+
+/***
+   KeymanWeb 10.0
+   Copyright 2017 SIL International
+***/
```

Comment:
This is a little outdated; see https://github.com/keymanapp/keyman/wiki/Keyman-Code-Style-Guide#File-header